src/api-v1.js

```diff
@@ -740,26 +740,64 @@ api.declare({
   route: '/state/:workerType',
   name: 'state',
   title: 'Get AWS State for a worker type',
-  scopes: [['aws-provisioner:view-worker-type:<workerType>']],
+  output: 'state-response.json#',
```

Comment:
Instead of getting rid of the scope-protected endpoint, I'd much rather that we add this as a new endpoint called `/simple-state` or something like that.  It would be a really unfortunate outcome to lose the ability to see and link to ec2 console for instances and other things in the UI that the current implementation of `/aws-state/` and `/state/` allow us to have.

As this method is implemented right now, I think it's great!